views/js/test/review/plugins/navigation/review-panel/panel-data/review-data-correct-with-score.json

```diff
@@ -15,16 +15,19 @@
               "label": "Example_0_Introduction",
               "informational": true,
               "skipped": false,
+              "withScore": true,
```

Comment:
Why we need this params 'withScore' in every item? 
As I understand in logic withScore === config.showScore from reviewPanel and we don't use withScore from item

Reply:
This is required by the template, especially on every item: https://github.com/oat-sa/extension-tao-review/pull/33/files#diff-35469fd0b5d24cb8f788779c02958a62R14

The bonus is the flag is also reported on upper levels for consistency (section, parts, test...)